### DISCUSSION TITLE
posts/game/2024/11/16/02/%EB%B8%94%EB%A3%A8%EC%95%84%EC%B9%B4-%EB%A6%AC%EA%B2%9C/

### DISCUSSION THREAD

**Discussion by giscus** [melted-mint/fuwari-edited#5] [Comments]
# posts/game/2024/11/16/02/%EB%B8%94%EB%A3%A8%EC%95%84%EC%B9%B4-%EB%A6%AC%EA%B2%9C/

까먹다가 올리는거.

https://meltedmint.com/posts/game/2024/11/16/02/%EB%B8%94%EB%A3%A8%EC%95%84%EC%B9%B4-%EB%A6%AC%EA%B2%9C/

**Comment by fgghf405** [comment]
고수;;
